modules/core/shared/src/main/scala/gem/Dataset.scala

```diff
@@ -58,6 +57,24 @@ object Dataset {
       Show.fromToString
 
   }
+  trait LabelOptics {
+    object Optics {
+
+      /** Format from Strings into Label and back. */
+      val fromString: Format[String, Label] =
```

Comment:
Why not putting `fromString` directly into `LabelOptics`?

Reply:
I kind of like having the `Optics` namespace for these.  I'm not sure I have a strong opinion on keeping the delegating `fromString` and `format` methods.  Part of me wants to get rid of them to avoid having all that extra code. Part of me thinks they are convenient and more intuitive than  calls that end in `reverseGet` and `getOption`.   I guess we'd get used to it though.  If forced to decide I guess I would remove the delegating methods and just put the `Optics` module directly in the companion instead of in a separate `trait`.